### DISCUSSION TITLE
Priorisierung der Eingänge am WR

### DISCUSSION THREAD

**Discussion by steeed** [hoylabs/OpenDTU-OnBattery#1470] [General]
Hi zusammen,

ich hab schon viel gesucht, aber nichts dazu gefunden.
Gibt es eine Möglichkeit beispielsweise 2 Eingänge am WR zu priorisieren? Oder ist Ähnliches geplant?

Zum Hintergrund: Durch das Firmware Update für die HMS Reihe von Hoymiles werden hier jetzt nicht mehr alle Eingänge gleichmässig limitiert, sondern der Ausgang.
Wenn man jetzt ein Setup hat, oder plant, wo beispielsweise 4 Panels an einem SolarFlow hängen und der SolarFlow an 2 Eingängen am WR hängt. Zusätzlich noch 2 Panels direkt am WR. 
Jetzt müsste nur irgendwie sichergestellt werden, dass die Leistung erstmal soweit wie möglich aus den beiden direkt angeschlossenen Panels kommt und nicht aus dem Speicher.

Zum Teil könnte man sich hier zu Nutze machen, dass der SolarFlow Hub etwas träge reagiert. Aber auch nur zum Teil.
Sagen wir der Verbrauch im Haus liegt bei 300W, Panels könnten 600W liefern, Speicher wird mit 1,4kW geladen.
Steigt jetzt der Verbrauch im Haus auf 600W, könnten die beiden Panels das decken und werden sie vermutlich auch, da der Hub die Leistung nur träge steigert.
Fällt der Bedarf im Haus, vermute ich nun das umgekehrte Verhalten. Da der Hub die Leistung auch nur Träge wieder verringert werden hier zuerst die beiden Panels limitiert, obwohl sie den Bedarf decken könnten.

Hier wäre eine Priorisierung der Eingänge sehr hilfreich um dieses Problem zu umgehen.

Setup aus meinem Beispiel:
SolarFlow Hub 2000 mit 4 Panels(2x425W Süd; 2x450W Südwest), 3,8kWh Speicher, Hoymiles HMS-1600-4T, Shelly 3EM Pro, openDTU OnBattery

**Comment by spcqike** [comment]
kurz um: das geht nicht.

man hat keinen Einfluss darauf, woher der Wechselrichter die Energie nimmt. Wenn du die Batterie nicht nehmen willst, musst du sie deaktivieren. In ihrer App oder extern per Hardware (Einschaltstrom beachten beim regelmäßigen Trennen)

alternativ:
nimm einen zweiten Wechselrichter her, einer mit PV, einer mit Batterie.
